parsers/to-react-native/README.md

```diff
@@ -30,8 +30,9 @@ interface parser {
       tabWidth: number;
       useTabs: boolean;
     }>;
-    formatFileName: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase' | 'none';
-    formatKeys: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase';
+    formatFileName?: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase' | 'none';
+    formatKeys?: 'camelCase' | 'kebabCase' | 'snakeCase' | 'pascalCase';
+    isTypescript?: boolean;
```

Comment:
Since parsers need to be evolved for the long run, we recommend the use of an object literal instead of boolean, so that future settings can be expressed.

```suggestion
    typescript?: {
      castToConst: true;
    }
```

Reply:
@nclsndr Thanks for this, how should I handle the decision for using `'typescript'` instead of `'babel'` in this case? My knee-jerk solution is to check if `options.typescript` is an object, but I don't like to infer based on object presence. Thoughts?

Reply:
I could just check against `castToConst` anyways, since that is *why* I'm using `'typescript'` over `'babel'`, but that seems like it might cause issues down the line.

Reply:
Switched to `babel-ts`, which properly handles both TypeScript and JavaScript.